addons/polardbx/templates/clusterDefinition.yaml

```diff
@@ -1,707 +0,0 @@
-apiVersion: apps.kubeblocks.io/v1alpha1
```

Comment:
Add ClusterDefinition & ComponentVersion chart for cloud integration

Reply:
It's weird to introduce the cd & cmpv CR but without necessary content to define within them.